tests/test_forms.py

```diff
@@ -0,0 +1,66 @@
+from unittest import TestCase
+
+from taxi.forms import (
+    DriverCreationForm,
+    DriverUsernameSearchForm,
+    CarModelSearchForm,
+    ManufacturerNameSearchForm
+)
+
+
+class DriverFormTest(TestCase):
+    def setUp(self) -> None:
+        self.form_data = {
+            "username": "petro",
+            "first_name": "Petro Chur",
+            "last_name": "Petro Chur Petrovych",
```

Comment:
The `first_name` and `last_name` fields in the `form_data` dictionary seem to contain full names instead of just the first and last names respectively. Ensure that `first_name` contains only the first name and `last_name` contains only the last name to match the expected field usage.